cmd/spec/path.yaml

```diff
@@ -280,6 +282,14 @@ paths:
             type: string
           required: true
           description: Hyperscaler region
+        - in: query
+          name: token
+          schema:
+            type: string
+            default: ""
+          required: false
+          description: The token to request the next page of results, empty token for first page.
+        - $ref: '#/components/parameters/Limit'
```

Comment:
I suggest to also create a component for token, we only have it once for now but this can prevent copy pasting.